<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widgets_ViewScenario</class>
 <widget class="QWidget" name="Widgets_ViewScenario">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="resultPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>392</width>
        <height>556</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Map Plotting</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="resultFileLabel">
         <property name="text">
          <string>Select a file to plot</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="fileList"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="toolTip">
            <string>If simulator output does not contain a value at a given point, this value is used</string>
           </property>
           <property name="text">
            <string>Fill Value</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="fillValueLineEdit">
           <property name="toolTip">
            <string>If simulator output does not contain a value at a given point, this value is used</string>
           </property>
           <property name="text">
            <string>0</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_11">
           <property name="toolTip">
            <string>If simulator output does not contain a value at a given point, this value is used</string>
           </property>
           <property name="text">
            <string>minValue</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLineEdit" name="minValue">
           <property name="toolTip">
            <string>Use this as the minimum value (deep blue) for the plot, all lower values will be mapped to this.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_12">
           <property name="toolTip">
            <string>If simulator output does not contain a value at a given point, this value is used</string>
           </property>
           <property name="text">
            <string>maxValue</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="maxValue">
           <property name="toolTip">
            <string>Use this as the maximum value (dark red) for the plot, all higher values will be mapped to this.</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Draw contour plot</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1" colspan="2">
          <widget class="QCheckBox" name="contourPlotCheckBox">
           <property name="layoutDirection">
            <enum>Qt::RightToLeft</enum>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>261</width>
           <height>171</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="redrawButton">
         <property name="text">
          <string>Redraw</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
